### PR TITLE
Create insert

### PR DESCRIPTION
**Pull request by MartinLBeacham**
### Description

Add insert.md as a New Term for Existing Concept Entry.

### Issue Solved

Closes [#4521](https://github.com/Codecademy/docs/issues/4521).

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/pandas/concepts/dataframe/terms/insert/insert.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+Output:
+
+```
+   grade  grade                name          home     alignment
+0     99     54           Jarnathan     Frozenfar   Lawful Good
+1     76    100              Sofina          Thay  Neutral Evil
+2     64     26        Edgin Darvis  Icewind Dale  Chaotic Good
+3     83     70  Simon the Sorcerer       Triboar  Chaotic Good
+```
```
Comment (mamtawardhani) [inline content/pandas/concepts/dataframe/terms/insert/insert.md:109]: Please add the printed statement "Further Updated Test Results:" at the start of the code.

File: content/pandas/concepts/dataframe/terms/insert/insert.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+## Codebyte Example (if applicable)
```
Comment (mamtawardhani) [inline content/pandas/concepts/dataframe/terms/insert/insert.md:111]:
```suggestion
## Codebyte Example
```

File: content/pandas/concepts/dataframe/terms/insert/insert.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+testResults = pd.DataFrame(data)
+ 
+# Print the original DataFrame
+print("Original Test Results:")
+print(testResults)
+```
+
+Output:
+
+```
```
Comment (mamtawardhani) [inline content/pandas/concepts/dataframe/terms/insert/insert.md:59]:
```suggestion
```shell
```

File: content/pandas/concepts/dataframe/terms/insert/insert.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+|    `column`     | str.                        | Sets the label for the new column.                                                                                                                                                                                                                                                                                          |
+|     `value`     | Scalar, Series, or array-like.                        | Content to be inserted as the new column.  Note that passing a single value will set that same value for all rows.                                                                                                                                                 |
+| `allow_duplicates` | bool. Default `False` | If `allow_duplicates` is set to `True`, the new column is allowed to have the same `column` value as a preexisting column. If `allow duplicates` is set to `False`, passing a column name that is associated with a preexisting column will throw a `ValueError`. |
+
+## Example
+
+In the following examples, the `.insert()` method is used with different parameters on the testResults DataFrame:
+
+```
```
Comment (mamtawardhani) [inline content/pandas/concepts/dataframe/terms/insert/insert.md:42]:
```suggestion
```py
```

File: content/pandas/concepts/dataframe/terms/insert/insert.md [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+   grade                name          home
+0     99           Jarnathan     Frozenfar
+1     76              Sofina          Thay
+2     64        Edgin Darvis  Icewind Dale
+3     83  Simon the Sorcerer       Triboar
+```
+
+Doric forgot to insert the alignment of each person tested.  She uses `insert()` to add the alignment of each person as follows:
+
+```
```
Comment (mamtawardhani) [inline content/pandas/concepts/dataframe/terms/insert/insert.md:70]:
```suggestion
```py
```

File: content/pandas/concepts/dataframe/terms/insert/insert.md [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+testResults.insert(3, 'alignment', ['Lawful Good','Neutral Evil','Chaotic Good','Chaotic Good'])
+
+#print  the updated DataFrame
+print("Updated Test Results:")
+print(testResults)
+```
+
+Output:
+
+```
```
Comment (mamtawardhani) [inline content/pandas/concepts/dataframe/terms/insert/insert.md:81]:
```suggestion
```shell
```

File: content/pandas/concepts/dataframe/terms/insert/insert.md [hunk 7]
```diff
@@ -0,0 +1,132 @@
...
+   grade                name          home     alignment
+0     99           Jarnathan     Frozenfar   Lawful Good
+1     76              Sofina          Thay  Neutral Evil
+2     64        Edgin Darvis  Icewind Dale  Chaotic Good
+3     83  Simon the Sorcerer       Triboar  Chaotic Good
+```
+
+Szass Tam used *scrying* to learn Doric's password.  To confuse Doric, Szass decided to add an identical *grade* column as follows:
+
+```
```
Comment (mamtawardhani) [inline content/pandas/concepts/dataframe/terms/insert/insert.md:92]:
```suggestion
```py
```

File: content/pandas/concepts/dataframe/terms/insert/insert.md [hunk 8]
```diff
@@ -0,0 +1,132 @@
...
+testResults.insert(1, 'grade', [54,100,26,70],allow_duplicates=True)
+
+#print  the updated DataFrame
+print("Further Updated Test Results:")
+print(testResults)
+```
+
+Output:
+
+```
```
Comment (mamtawardhani) [inline content/pandas/concepts/dataframe/terms/insert/insert.md:103]:
```suggestion
```shell
```

File: content/pandas/concepts/dataframe/terms/insert/insert.md [hunk 9]
```diff
@@ -0,0 +1,132 @@
...
+   grade  grade                name          home     alignment
+0     99     54           Jarnathan     Frozenfar   Lawful Good
+1     76    100              Sofina          Thay  Neutral Evil
+2     64     26        Edgin Darvis  Icewind Dale  Chaotic Good
+3     83     70  Simon the Sorcerer       Triboar  Chaotic Good
+```
+
+## Codebyte Example (if applicable)
+
+Feel free to play around with Doric's `testResults` DataFrame and add your own column.  Some suggestions include: test date(s), each person's class (such as 'Rogue' or 'Barbarian'), or known associates.
```
Comment (mamtawardhani) [inline content/pandas/concepts/dataframe/terms/insert/insert.md:113]:
```suggestion
The following example demonstrates the use of the .insert() method in pandas to add a new column named 'alignment' to a DataFrame, representing alignment information for test results.```

File: content/pandas/concepts/dataframe/terms/insert/insert.md [hunk 10]
```diff
@@ -0,0 +1,132 @@
...
+
+```codebyte/js
```
Comment (mamtawardhani) [inline content/pandas/concepts/dataframe/terms/insert/insert.md:115]:
```suggestion
```codebyte/python
```

File: content/pandas/concepts/dataframe/terms/insert/insert.md [hunk 11]
```diff
@@ -0,0 +1,132 @@
...
+|    `column`     | str.                        | Sets the label for the new column.                                                                                                                                                                                                                                                                                          |
+|     `value`     | Scalar, Series, or array-like.                        | Content to be inserted as the new column.  Note that passing a single value will set that same value for all rows.                                                                                                                                                 |
+| `allow_duplicates` | bool. Default `False` | If `allow_duplicates` is set to `True`, the new column is allowed to have the same `column` value as a preexisting column. If `allow duplicates` is set to `False`, passing a column name that is associated with a preexisting column will throw a `ValueError`. |
+
+## Example
+
+In the following examples, the `.insert()` method is used with different parameters on the testResults DataFrame:
```
Comment (mamtawardhani) [inline content/pandas/concepts/dataframe/terms/insert/insert.md:40]:
```suggestion
In the following example, the `.insert()` method is used with different parameters on the testResults DataFrame:
```

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Hey @MartinLBeacham 

Thank you for your contribution at Codecademy!
The entry was nicely written!

I've mentioned a few comments and suggestions for this entry, could you please review them and make the required changes at your earliest convenience? 
Thanks!

**Comment by MartinLBeacham** [comment]
Thank you so much @mamtawardhani !

I've committed your edits and incorporated your comments into an updated commit.

Please let me know if you have any further comments or questions.

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Hey @MartinLBeacham 

Just one more suggestion added, please modify the same.
Thanks!

**Review by Sriparno08 — APPROVED** [review]
Looks good 👍

**Pull request merged**